VenafiPS/Public/Get-TppIdentity.ps1

```diff
@@ -119,6 +134,12 @@ function Get-TppIdentity {
                         $response | Add-Member @{ 'Associated' = $associated.Identities }
                     }
 
+                    if (($response.IsGroup) -and ($GetMembers))  {
+                        $params.UriLeaf = 'Identity/GetMembers'
+                        $params.Body.Add("ResolveNested","1");
+                        $members = Invoke-VenafiRestMethod @params
+                        $response | Add-Member @{ 'Members' = $members.Identities}
```

Comment:
```suggestion
                        $response | Add-Member @{ 'Member' = $members.Identities}
```

Reply:
It is an array with a number of members, shouldn't it be plural ?